example/Makefile

```diff
@@ -41,13 +41,17 @@ run:
 	$(HELM) repo add jetstack https://charts.jetstack.io
 	$(HELM) repo update
 	$(HELM) dependency build ../charts/topolvm/
-	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f ./values.yaml
+	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f ./values.yaml --set image.tag=$(TOPOLVM_VERSION)
```

Comment:
We test the chart with the default image version to ensure it works as-is. We will release a new chart soon. Please wait a moment.

Reply:
Ack, this tripped me off so badly. Will wait then.